assets/json/campsitesfinal.json

```diff
@@ -8650,10 +8650,10 @@
   {
     "url": "https://www.facebook.com/groups/free.code.camp.suwon",
     "city": "Suwon",
-    "state": "",
+    "state": "Gyeonggi-do",
     "country": "South Korea",
     "coordinates": "33.429441, 126.267859",
-    "photoUrl": ""
+    "photoUrl": "https://user-images.githubusercontent.com/12888144/27015187-38d3456a-4f43-11e7-83d3-1bdc810def3c.jpeg"
```

Comment:
Not to nitpick, but it'd be preferable if the image was hosted on Facebook (your group's album, maybe?)